Library/Homebrew/livecheck/strategy/crate.rb

```diff
@@ -0,0 +1,115 @@
+# typed: true
+# frozen_string_literal: true
+
+module Homebrew
+  module Livecheck
+    module Strategy
+      # The {Crate} strategy identifies versions of a crates.io package by
+      # checking the information from the `versions` API endpoint.
```

Comment:
Nit: For consistency, should we use the registry name (i.e., crates)?

Reply:
After thinking about it some more earlier, I've been leaning towards `Crates` as the strategy name, since it does seem to be more in line with our existing strategy names. I'll update the name on the next push.